attorch/module.py

```diff
@@ -23,9 +23,17 @@ def __len__(self):
         return len(self._modules)
 
     def __iter__(self):
-        return iter(self._modules.values())
+        return iter(self._modules.keys())
 
     def keys(self):
         yield from self._modules.keys()
 
 
+    def items(self):
+        return self._modules.items()
+
+    def values(self):
+        return self._modules.values()
+
+    def keys(self):
```

Comment:
I'd say go ahead and delete the first version of `keys` - use `return` instead of `yield from` as you have done.